script.js

```diff
@@ -161,19 +161,19 @@ let main = async function () {
         
         let postsToDelete = filterFirstXPosts(c, HowManyRecentPostsToKeep);
 
-        if (postsToDelete.length < 1 || i > 50) {
+        if (postsToDelete.length < 1 || i > 50000) {
             console.log("Done, reached limit", postsToDelete.length, i);
             break;
         }
 
-        await delay(3000); // So you can see the keeping table quickly
+        await delay(8000); // So you can see the keeping table quickly
         
         console.table(postsToDelete, ["date", "author", "text"]);
 
         //deletePost(postsToDelete[1].node); // To delete a specific post, for testing only
-        //await deletePosts(postsToDelete); // Uncomment to this to delete all posts other that HowManyRecentPostsToKeep first
+        await deletePosts(postsToDelete); // Uncomment to this to delete all posts other that HowManyRecentPostsToKeep first
```

Comment:
This is intentionally commented so people who are not coders who try to run this script don't delete things unintentionally. It requires the person to understand what they are doing before running the script (you are a person who understands as you saw this as a problem). In the script help instructions it says to uncomment this then ready to delete.